integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/support/ws/core/CoreDeviceManagementClient.java

```diff
@@ -149,4 +151,11 @@ public UpdateDeviceCdmaSettingsResponse getUpdateDeviceCdmaSettingsResponse(
         return (UpdateDeviceCdmaSettingsResponse) wst.marshalSendAndReceive(asyncRequest);
     }
 
+    public UpdateDeviceResponse updateDevice(final UpdateDeviceRequest request)
+            throws WebServiceSecurityException, GeneralSecurityException, IOException {
```

Comment:
Can we change this somehow, so we throw only one exception? Sonar checks "Public methods should throw at most one checked exception".